### PR TITLE
Explicit modulemap added

### PR DESCRIPTION
**Pull request by olmisa**
* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] ~Did you add unit tests?~
* [x] Did you test your change with the sample apps?

## Description

To enable non-SPM building the explicit modulemap provided.

## Related PRs or issues

[AB#80641](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/80641)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # PLCrashReporter Change Log
 
+## Version 1.7.2 (Under development)
+
+* Fix `xcodebuild` couldn't find modulemap if PLCrashReporter integrated vid Swift Package Manager.
```
Comment (MatkovIvan) [inline CHANGELOG.md:5]: vid -> via

Please be more specific, it's not clear what scenarious exactly are affected.

File: include/module.modulemap [hunk 2]
```diff
@@ -0,0 +1,8 @@
+framework module CrashReporter {
```
Comment (MatkovIvan) [inline include/module.modulemap:1]: Use links like other files in this directory, no need to copy.

BTW why it should be in `include`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # PLCrashReporter Change Log
 
+## Version 1.7.2 (Under development)
+
+* Fix `xcodebuild` command without `scheme` parameter couldn't find modulemap if PLCrashReporter integrated via Swift Package Manager.
```
Comment (annakocheshkova) [inline CHANGELOG.md:5]:
```suggestion
* Fix `xcodebuild` command without `scheme` parameter couldn't find modulemap if **PLCrashReporter** is integrated via Swift Package Manager.
```

Comment (Jamminroot) [inline CHANGELOG.md:5]: Probably a nitpick, but I'd simplify it a bit 
```suggestion
* Fix `xcodebuild` command execution without `scheme` parameter with PLCrashReporter integrated via Swift Package Manager.
```

File: include/module.modulemap (hunk 2 above)
Comment (olmisa) [reply include/module.modulemap]: I tried other places in test branches but have no success. Also, I have found during investigation that people were faced with the error _modulemap '/some/path/module.modulemap' should be inside the 'include' directory_.

**Pull request closed without merging**